### PR TITLE
Add pre-reqs for OpenShift

### PR DESCRIPTION
**Pull request by sdodsley**
This PR adds documentation for OpenShift to ensure that multipath and iSCSI are correctly configured and enabled on worker nodes.

It adds the recommended multipath configuration information for Pure devices as well as applying the recommended udev settings

These setting are for RHEL7 and RHCOS.

when RHEL8 is supported by OpenShift these will need to be updated

### CODE REVIEW

File: docs/99-pure-storage-attach.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  config:
+    ignition:
+      version: 2.2.0
+    storage:
+      files:
+      - path: /etc/multipath.conf
+        mode: 384
+        filesystem: root
+        contents:
+          source: data:,devices%20%7b%0a%20%20device%20%7b%0a%20%20%20%20%20%20%20%20vendor%20%22PURE%22%0a%20%20%20%20%20%20%20%20product%20%22FlashArray%22%0a%20%20%20%20%20%20%20%20fast_io_fail_tmo%2010%0a%20%20%20%20%20%20%20%20path_grouping_policy%20%22group_by_prio%22%0a%20%20%20%20%20%20%20%20failback%20%22immediate%22%0a%20%20%20%20%20%20%20%20prio%20%22alua%22%0a%20%20%20%20%20%20%20%20hardware_handler%20%221%20alua%22%0a%20%20%20%20%20%20%20%20max_sectors_kb%204096%0a%20%20%20%20%7d%0a%7d
```
Comment (dsupure) [inline docs/99-pure-storage-attach.yaml:17]: Do we want to hard-code the `multipath.conf` setting here?
We have a script that they can manually run on each worker nodes: https://github.com/dsupure/multipath_tool/tree/master/scripts/multipath-utils
Users can customized their settings.

File: docs/openshift_mc.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# Pre-Requisites for OpenShift 4.4 and higher
+
+## Preparing worker nodes
+Perform these steps before installing PSO:
+
+### 1. Perform this step to ensure iSCSI connectivity, when using RHEL OS, for each worker node.
+If using RHCOS or if the packages are already installed, continue to the next step.
+
+RHEL 7.x:
```
Comment (carysyd) [inline docs/openshift_mc.md:9]: Is there any particular reason only `RHEL 7.x` is listed here? Or maybe RHEL8 has these packages installed by default?

File: docs/openshift_mc.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+The [99-pure-storage.yaml](./99-pure-storage.yaml) file is for both Fibre Channel and iSCSI configurations:
+
+
+**Important:** The  `99-pure-storage-attach.yaml` configuration file overrides any files that already exist on your system. Only use this file if the files mentioned in the yaml below are not already created. If one or more have been created, edit this yaml file, as necessary.
+
+Apply the yaml file.
+
+```bash
+oc apply -f 99-pure-storage-attach.yaml
```
Comment (carysyd) [inline docs/openshift_mc.md:26]: minor suggestion:
If we expect most users would just apply the YAML file directly without modifying it, maybe we could use the github raw content URL for this file. It's easy for user to apply.

**Review by carysyd — APPROVED** [review]
LGTM, only two minor suggestions/questions.

File: docs/openshift_mc.md (hunk 3 above)
Comment (sdodsley) [reply docs/openshift_mc.md:26]: Yes - I can change that, but now I have to think about support for RHEL 8 as well as the multipath and udev settings are different

File: docs/openshift_mc.md (hunk 2 above)
Comment (sdodsley) [reply docs/openshift_mc.md:9]: Good point - I will have to think about this a bit more.

**Review by dsupure — APPROVED** [review]
My question about the multipath.conf is answered on the slack channel. LGTM!

File: docs/openshift_mc.md (hunk 3 above)
Comment (sdodsley) [reply docs/openshift_mc.md]: It appears that I have the correct settings now for the supported RHEL versions

**Pull request merged**